draft-ietf-webpush-protocol.xml

```diff
@@ -414,10 +416,9 @@ Link: </set/4UXwi2Rd7jGS7gp5cuutF8ZldnEuvbOy>;
 ]]></artwork>
       </figure>
         <t>
-          This gives the push service the option to create a subscription within
-          an existing subscription set.  The push service returns the same
-          subscription set in the response, though it MAY use a new subscription
-          set if it is unable to reuse the provided one.
+          The push service returns the same subscription set in its response, although

```

Comment:
Can we say  "SHOULD return the same subscription set", just to make this a little stronger?